controllers/grants/index.js

```diff
@@ -150,85 +143,82 @@ async function checkSpelling(searchTerm, locale = 'en') {
 }
 
 router.get('/', injectHeroImage('tinylife'), injectCopy('funding.pastGrants.search'), async (req, res, next) => {
-    let data;
-    const facetParams = buildAllowedParams(req.query);
-    const paginationLabels = req.i18n.__('global.misc.pagination');
-    let queryWithPage = addPaginationParameters(facetParams, req.query.page);
-    queryWithPage.locale = res.locals.locale;
-
     try {
-        data = await queryGrantsApi(queryWithPage);
+        const locale = req.i18n.getLocale();
```

Comment:
Inlined this code all into the try/catch block. Not 100% how the async behaviour acts if we have one path of code in the try/catch and the other outside it. Moves this all into the main block.